released/packages/coq-pi-agm/coq-pi-agm.1.2.5/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+name: "coq-pi-agm"
+version: "1.2.5"
+maintainer: "yves.bertot@inria.fr"
+
+homepage: "http://www-sop.inria.fr/members/Yves.Bertot/"
+bug-reports: "yves.bertot@inria.fr"
+license: "CeCILL-B"
+build: [["coq_makefile" "-f" "_CoqProject" "-o" "Makefile" ]
+       [ make "-j" "%{jobs}%" ]]
+install: [ make "install" "DEST='%{lib}%/coq/user-contrib/pi_agm'" ]
```

Comment:
Is there an obvious reason for the logpath (`agm`) to diverge from the installed directory name (`pi_agm`)? To my knowledge, this means that all modules will have to be accessed by consumers of the library using `pi_agm`.